src/containers/flows/MintDropNFT/labels.tsx

```diff
@@ -55,56 +76,26 @@ export const getBannerLabel = (
 			);
 		}
 	}
-	if (dropStage === Stage.Whitelist) {
-		let timer;
-		if (countdownDate && !isFinished) {
-			timer = (
-				<span style={{ display: 'inline-block', marginTop: 6 }}>
-					Available to everyone in{' '}
-					{countdownDate && (
-						<Countdown to={countdownDate} onFinish={onFinish} />
-					)}
-				</span>
-			);
-		}
-		if (countdownDate && isFinished) {
-			timer = (
-				<span style={{ display: 'inline-block', marginTop: 4 }}>
-					Public release starting now - waiting for contract to begin
-				</span>
-			);
-		}
-
-		return (
-			<div style={{ display: 'flex', flexDirection: 'column' }}>
-				<span>
-					Wilder Pets now available for whitelisted supporters{' '}
-					{totalLabel(wheelsMinted!, wheelsTotal!)}
-				</span>
-				{timer}
-			</div>
-		);
-	}
 	if (dropStage === Stage.Public) {
-		if (Math.max(wheelsTotal! - wheelsMinted!, 0) !== 0) {
-			return (
-				<>
-					The pets sale is finished. {Math.max(wheelsTotal! - wheelsMinted!, 0)}{' '}
-					remaining pets have been transferred to the Wilder DAO.
-				</>
-			);
-		}
+		// if (Math.max(wheelsTotal! - wheelsMinted!, 0) !== 0) {
+		// 	return (
+		// 		<>
+		// 			The beasts sale is finished.{' '}
+		// 			{Math.max(wheelsTotal! - wheelsMinted!, 0)} remaining beasts have been
+		// 			transferred to the Wilder DAO.
+		// 		</>
+		// 	);
+		// }
 		return (
 			<>
-				Minting is now open to everyone, act fast to secure your Pets!{' '}
+				Minting is now open to everyone, act fast to secure your Beasts!{' '}
 				{totalLabel(wheelsMinted!, wheelsTotal!)}
 			</>
 		);
 	}
 
 	if (dropStage === Stage.Sold) {
-		return <>All {wheelsTotal} Pets have been minted</>;
-		// return <>Air Wild Season One is sold out!</>;
+		return <>Wolves Sale is Complete</>;
```

Comment:
1. Noticed we used "Wolves" here, but "Beasts" everywhere else. Should change to "Beasts" unless explicitly specified.
2. Should change the capitalisation to `Wolves sale is complete` unless specified otherwise.

Reply:
I think we followed messaging shared by Wilders Team